### PR TITLE
Navigation 

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 60px;
+}
+.header{
+    margin-top: 20px;
+    display: flex;
+    width: 735px;
+    height: 61px;
```
Comment (Naveenchandar) [inline components/Navigation/navigation.css]: Any reason behind using px instead of relative units ?

File: components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+}
+button:hover{
+    cursor: pointer;
+}
+.actionbtn{
+    margin-left: auto;
+}
+span{
+    color: white;
```
Comment (Naveenchandar) [inline components/Navigation/navigation.css:39]: Please use hex code for consistency.

File: components/Navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    border: none;
+
+}
+button:hover{
+    cursor: pointer;
+}
+.actionbtn{
+    margin-left: auto;
+}
+span{
```
Comment (Naveenchandar) [inline components/Navigation/navigation.css:38]: Try to use other selectors. It is too generic and may be it'll affect in all other places.

File: components/Navigation/navigation.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    padding: 12px;
+}
+.headertxt1{
+    padding: 13px;
+}
+
+
+
+
+
```
Comment (Naveenchandar) [inline components/Navigation/navigation.css]: Please remove empty spaces

File: components/Navigation/navigation.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <div class="actionbtn1">
+            <div class="btn"><button><i class= "fa fa-download">	</i></button>
+                <button><i class="fa fa-print">	</i></button>
+                <button><i class="fa fa-bookmark">	</i></button></div>
+            
+        
+        </div>
+        </div>
+        
+
```
Comment (Naveenchandar) [inline components/Navigation/navigation.html]: Indentation missing. Please format the code

**Review by Naveenchandar — CHANGES_REQUESTED** [review]
Please convert all 'px' to relative units.

File: components/Navigation/navigation.css (hunk 2 above)
Comment (jeetbhalani15) [reply components/Navigation/navigation.css:39]: done

File: components/Navigation/navigation.css (hunk 1 above)
Comment (jeetbhalani15) [reply components/Navigation/navigation.css]: done

File: components/Navigation/navigation.html (hunk 5 above)
Comment (jeetbhalani15) [reply components/Navigation/navigation.html]: dsone

File: components/Navigation/navigation.css [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    width: 45rem;
+    height: 3rem;
+    background-color: #6200EE;
+    align-items: center;
+   
+}
+button{
+    width: 2rem;
+    height: 2rem;
+    background-color: #6200EE;
```
Comment (ishitakeshawani) [inline components/Navigation/navigation.css:25]: define a variable for color and reuse it.

File: components/Navigation/navigation.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        <div class="actionbtn">
+          <div class="btn">
+            <button><i class="fa fa-download"> </i></button>
+            <button><i class="fa fa-print"> </i></button>
+            <button><i class="fa fa-bookmark"> </i></button>
+          </div>
+        </div>
+      </div>
+      <div class="header1">
```
Comment (ishitakeshawani) [inline components/Navigation/navigation.html]: What is the difference between the two headers? It seems the same.

File: components/Slider/slider.css [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    background-color: white;
+    padding: 20px 20px 20px 30px;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, .1);
+    display: flex;
+    align-items: center;
+}
+ .box .slider{
+    height: 40px;
+    width: 300px;
```
Comment (ishitakeshawani) [inline components/Slider/slider.css:17]: use rem

File: components/Navigation/navigation.html (hunk 7 above)
Comment (ishitakeshawani) [inline components/Navigation/navigation.html]: Try not to use numbers in class name.

**Review by ishitakeshawani — CHANGES_REQUESTED** [review]

**Review by Mahi24-prog — APPROVED** [review]
Make it responsive and use rem instead of px

**Review by Jay-Gadhiya — APPROVED** [review]

**Pull request merged**